tests/test_api.rs

```diff
@@ -1553,6 +1554,49 @@ fn object() {
     assert!(!object.has(scope, n_unused).unwrap());
     assert!(object.delete(scope, n1.into()).unwrap());
     assert!(!object.has(scope, n1.into()).unwrap());
+
+    let global = context.global(scope);
+    let object_string = v8::String::new(scope, "o").unwrap().into();
+    global.set(scope, object_string, object.into());
+
+    let true_value = v8::Boolean::new(scope, true);
+    let false_value = v8::Boolean::new(scope, false);
+
+    assert_eq!(eval(scope, "Object.isExtensible(o)").unwrap(), true_value);
+    assert_eq!(eval(scope, "Object.isSealed(o)").unwrap(), false_value);
+    assert_eq!(eval(scope, "Object.isFrozen(o)").unwrap(), false_value);
```

Comment:
It's slightly more idiomatic (and shorter) to use is_true() and is_false():
```suggestion
    assert!(eval(scope, "Object.isExtensible(o)").unwrap().is_true());
    assert!(eval(scope, "Object.isSealed(o)").unwrap().is_false());
    assert!(eval(scope, "Object.isFrozen(o)").unwrap().is_false());
```
I wanted to suggest this as a diff for the whole test but somehow GH won't let me. At any rate, that's the only nit I have and it's not really a blocker, more a suggestion.